packages/plugin-ext/src/common/plugin-api-rpc-model.ts

```diff
@@ -153,7 +153,11 @@ export interface SingleEditOperation {
 }
 
 export interface Command {
+    /**
+     * @deprecated use command instead
+     */
     id: string;
+    command?: string;
```

Comment:
after looking closer, actually it has to be `id` here, because this command should be aligned with Monaco command: https://github.com/eclipse-theia/theia/blob/eae45daed3ab1c0b0861dc924fbc929f6d7021a3/node_modules/%40typefox/monaco-editor-core/monaco.d.ts#L5508-L5513

even though that vscode APIs are using `command` not `id`

I think we should fix typings here: https://github.com/eclipse-theia/theia/blob/eae45daed3ab1c0b0861dc924fbc929f6d7021a3/packages/plugin-ext/src/common/plugin-api-rpc.ts#L610-L612 

i.e.:
```ts
acceptInputCommand?: Command;
statusBarCommands?: Command[];
```
since we actually send plugin command not scm command and then do adjustment in scm-main